### PR TITLE
config: enable blockade and label for chaos-mesh/website and chaos-mesh/website

### PR DESCRIPTION
**Pull request by hi-rustin**
Enable blockade and label for chaos-mesh/website and chaos-mesh/website.

### CODE REVIEW

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -626,6 +626,28 @@ ti-community-label:
...
+      - 'help-wanted'
+      - 'docs'
+      - 'missing-translation-status'
+      - 'require-LGT1'
+      - 'require-LGT2'
+      - 'require-LGT3'
+      - 'requires-followup'
+    exclude_labels:
+      - status/can-merge
+
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:650]:
```suggestion
```

File: prow/config/plugins.yaml [hunk 2]
```diff
@@ -89,6 +89,18 @@ require_matching_label:
     prs: true
     regexp: (^translation/)|(type/cherry-pick-for-(.*)$)
     grace_period: 10s
+  - missing_label: missing-translation-status
+    org: chaos-mesh
+    repo: website
+    prs: true
+    regexp: (^translation/)|(type/cherry-pick-for-(.*)$)|(docs)
```
Comment (qiancai) [inline prow/config/plugins.yaml:96]:
```suggestion
    regexp: (^translation/)|(docs)
```

File: prow/config/plugins.yaml [hunk 3]
```diff
@@ -89,6 +89,18 @@ require_matching_label:
...
+    grace_period: 10s
+  - missing_label: missing-translation-status
+    org: chaos-mesh
+    repo: website-zh
+    prs: true
+    regexp: (^translation/)|(type/cherry-pick-for-(.*)$)|(docs)
```
Comment (qiancai) [inline prow/config/plugins.yaml:102]:
```suggestion
    regexp: (^translation/)|(docs)
```

**Review by qiancai — COMMENTED** [review]
type/cherry-pick-for label is not needed for the two repo, sorry I forgot that before.

**Pull request merged**